components/nimbus/ios/Nimbus/FeatureVariables.swift

```diff
@@ -332,7 +332,11 @@ internal class JSONVariables: VariablesWithBundle {
     }
     
     func asVariablesMap() -> [String: Variables]? {
-        return nil
+        let j = json.mapValues() {(value) in
+            // TODO: filter out any possible json that can't be cast to the dictionary
```

Comment:
What are you going to do with the json that can't be cast to a dictionary?

Reply:
It will be omitted! I can't think of a case where this can happen, but I went for the least destructive response if it does 😬